shell/platform/windows/platform_handler.cc

```diff
@@ -447,4 +448,14 @@ void PlatformHandler::HandleMethodCall(
   }
 }
 
+// static
+bool PlatformHandler::WindowProcCallback(HWND hwnd, UINT msg, WPARAM wpar, LPARAM lpar, void* user_data, LRESULT* result) {
+  PlatformHandler* that = static_cast<PlatformHandler*>(user_data);
+  if (msg == WM_CLOSE) {
```

Comment:
This blocks the window from closing, and assumes that closing the window is meant to close the app. What happens when we eventually have multiple windows, won't we have to check to make sure this is the last window before asking? Also, is there a case where the developer might want the window to actually close, but then keep running the app until they are ready to exit?

Reply:
It would require a check if it is the last window, but I do not think that check is possible to write yet.

Reply:
Could you use a Windows API to find how many open windows an app has? I think that even when we can enumerate Flutter-controlled windows in an app, we won't want to actually exit the app unless this is the last window, including windows that aren't Flutter controlled.